bundle/deploy/terraform/init.go

```diff
@@ -158,6 +158,18 @@ func inheritEnvVars(ctx context.Context, environ map[string]string) error {
 		}
 	}
 
+	// If there's a DATABRICKS_OIDC_TOKEN_ENV set, we need to pass the value of the environment variable defined in DATABRICKS_OIDC_TOKEN_ENV to Terraform.
+	// This is necessary to ensure that Terraform can use the same OIDC token as the CLI.
+	oidcTokenEnv, ok := env.Lookup(ctx, "DATABRICKS_OIDC_TOKEN_ENV")
+	if !ok {
+		oidcTokenEnv = "DATABRICKS_OIDC_TOKEN"
+	}
```

Comment:
If this is not the default, we need to pass through `DATABRICKS_OIDC_TOKEN_ENV` as well.

Otherwise TF won't find the right env var to use.